### ISSUE TITLE
Card subcomponents are missing `textAlign` props

### ISSUE THREAD

**Issue by levithomason** [Semantic-Org/Semantic-UI-React#2027]
Per SUI core, a Card's children can be aligned:

https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/card.css#L203

```jsx
<Card>
  <Card.Content textAlign="center">
    This should be centered.
  </Card.Content>
</Card>
```

This is true for all Card subcomponents.

**Comment by itamar244** [comment]
I am trying to do this change.
Should I implement this to every Card.* Component or do it somehow generally?
The propType is `PropTypes.oneOf(['left', 'center', 'right'])` (the is SUI.TEXT_ALIGNMENTS constant but in the css file you listed above the isn't a `justified` class)?

Also I'm a new contributor, can you please help me out with commit message?
i have problems with adding tests too. Need a little help.

**Comment by itamar244** [comment]
Solved the tests.
Do I need to add typescript definitions?
Also there isn't a type of `'left' | 'center' | 'right'`

**Comment by levithomason** [comment]
Awesome!  Go ahead an open a PR so folks know this is taken.  In your PR description, please note that it `Fixes #2027` so it auto closes this issue on merge 👍 

***

>Should I implement this to every Card.* Component

Yes, please.

>the is SUI.TEXT_ALIGNMENTS constant but in the css file you listed above the isn't a justified class

See Segment.js for example:

```jsx
PropTypes.oneOf(_.without(SUI.TEXT_ALIGNMENTS, 'justified')),
```

>Do I need to add typescript definitions? Also there isn't a type of 'left' | 'center' | 'right'.

Yes, please.  We are just listing them explicitly, see Table typings for example.

```ts
textAlign?: 'center' | 'left' | 'right';
```

**Comment by itamar244** [comment]
Opened a PR and finished changes.
